### PR TITLE
Internal: Knowledge Transfer - Testing widgets using ScreenShotter [ED-7057]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: docs/tests/screenshotter.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Screenshotter
+
+ScreenShotter automates visual regression testing of your responsive web UI by comparing DOM screenshots over time.
+
+This is an internal tool that's being used in Elementor's CI process, in order to make sure that the PR changes don't
+break the original behavior of the product.
```
Comment (matipojo) [inline docs/tests/screenshotter.md:6]:
```suggestion
This is an internal tool that's being used in Elementor's CI process, in order to make sure that the PR changes don't break the original behavior of the product.
```

File: docs/tests/screenshotter.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Features:
+- Install & Configure a WordPress environment automatically.
+- Set layout settings for print and screen.
```
Comment (matipojo) [inline docs/tests/screenshotter.md:10]: for print?

**Review by matipojo — CHANGES_REQUESTED** [review]
👍

File: docs/tests/screenshotter.md (hunk 1 above)
Comment (StyleShit) [reply docs/tests/screenshotter.md:6]: That's intentional, in order to keep a reasonable line length:
image

File: docs/tests/screenshotter.md (hunk 2 above)
Comment (StyleShit) [reply docs/tests/screenshotter.md:10]: It was copy pasted 😅 
removed

File: docs/tests/screenshotter.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- Simulate user interactions with Puppeteer scripts.
+
+## Usage:
+
+### How to Create a Template:
+1. Go to Elementor's "Templates" category in the wp-admin Dashboard.
+2. Click on "Add New".
+3. In the opened popup, select "Page" as type, and the name of the widget you're testing as a name.
+4. After the Editor is loaded, drag the widget you are testing, and add as many variations/combinations of controls as possible.
+5. Add an explanatory title (e.g. with the name of the control) that represents each block of control values/options/variations.
```
[the remark this answers is not in the document]
Comment (StyleShit) [reply docs/tests/screenshotter.md:21]: Done

**Pull request merged**